libevmasm/Assembly.cpp

```diff
@@ -74,6 +76,210 @@ unsigned Assembly::codeSize(unsigned subTagSize) const
 	}
 }
 
+void Assembly::importAssemblyItemsFromJSON(Json::Value const& _code, vector<string> const& _sourceList)
+{
+	solAssert(m_items.empty());
+	solRequire(_code.isArray(), AssemblyImportException, "Supplied JSON is not an array.");
+	for (auto current = begin(_code); current != end(_code); ++current)
+	{
+		auto const& item = m_items.emplace_back(createAssemblyItemFromJSON(*current, _sourceList));
+		if (item == Instruction::JUMPDEST)
+			solThrow(AssemblyImportException, "JUMPDEST instruction without a tag");
+		else if (item.type() == AssemblyItemType::Tag)
+		{
+			++current;
+			if (current != end(_code) && createAssemblyItemFromJSON(*current, _sourceList) != Instruction::JUMPDEST)
+				solThrow(AssemblyImportException, "JUMPDEST expected after tag.");
+		}
+	}
+}
+
+AssemblyItem Assembly::createAssemblyItemFromJSON(Json::Value const& _json, std::vector<std::string> const& _sourceList)
```

Comment:
```suggestion
AssemblyItem Assembly::createAssemblyItemFromJSON(Json::Value const& _json, vector<string> const& _sourceList)
```
There are also other occurrences like this where we can remove the `std` qualifier in `cpp` files.